pnpm-lock.yaml

```diff
@@ -3459,6 +3438,9 @@ packages:
   '@next/env@14.2.21':
     resolution: {integrity: sha512-lXcwcJd5oR01tggjWJ6SrNNYFGuOOMB9c251wUNkjCpkoXOPkDeF/15c3mnVlBqrW4JJXb2kVxDFhC4GduJt2A==}
 
+  '@next/env@14.2.23':
```

Comment:
Is this intended?

Reply:
Not intended at all, just rerun `pnpm install` to get fresh lockfile after removing the `@clerk/clerk-sdk-node` package 👀